webcompat/api/uploads.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'''Flask Blueprint for image uploads.'''
+
+import json
+
+from flask import abort
+from flask import Blueprint
+from flask import render_template
+from flask import request
+from flaskext.uploads import configure_uploads
+from flaskext.uploads import IMAGES
+from flaskext.uploads import patch_request_class
+from flaskext.uploads import UploadSet
+from flaskext.uploads import UploadNotAllowed
+from werkzeug import secure_filename
+
+from webcompat import app
+
+uploads = Blueprint('uploads', __name__, url_prefix='/upload',
+                    template_folder='../templates')
+JSON_MIME = 'application/json'
+images = UploadSet('uploads', IMAGES)
+configure_uploads(app, images)
+
+# limit image uploads to 4MB
+patch_request_class(app, 4 * 1024 * 1024)
+
```

Comment:
houla. I need a second pair of glasses. Thanks Mike. 
Just took a full screenshot of my tiny screen and it is ~400Ko PNG
The limit you set seems entirely reasonable.